<?xml version="1.0"?>

<launch>

  <node pkg="dead_reckoning" type="dead_reckoning" name="dead_reckoning" output="screen" >
    <param name="parent_frame" value="odom" />
    <param name="child_frame" value="roundbot/base_footprint" />
  </node>

</launch>
